oem/firstvoices/android/app/src/main/java/com/firstvoices/keyboards/SystemKeyboard.java

```diff
@@ -33,6 +33,8 @@
 import io.sentry.Sentry;
 
 public class SystemKeyboard extends InputMethodService implements OnKeyboardEventListener {
+    public static final String FV_BANNER_THEME_GOLD =
```

Comment:
What does the iOS FV app do for the banner?

Reply:
@jahorton or @sgschantz ?